### PR TITLE
Added more documentation on load balancer components

### PR DESCRIPTION
**Pull request by rbibby**

### CODE REVIEW

**Comment by rbibby** [comment]
@bluemorbo @Xiol another update to the load balancer docs to check through when you have chance. I've moved some stuff out of getting started in to individual pages for each screen so that I can update MyUKFast to point to them

File: source/network/load-balancers/components/access-controls.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* **Header matches** - Will be true if the specified header matches a certain value.
+* **URL begins with** - Will be true if the URL being visited begins with a certain value.
+
+## Available Actions
+
+* **Block traffic** - Stops the request from reaching the server, the user will get a 403 response.
+* **Redirect traffic** - Enter the full web address for the redirect location. Status codes should normally be 301 for a permanent change or 302 for a temporary redirect.
+* **Use target group** - Will send all requests to the specified target group which must be on the same load balancer.
+* **Set request header** - Adds the specified header with value to the requests going into the load balancer and being passed on to target servers.
+* **Set response header** -  Adds the specified header with value to the responses leaving the load balancer from the target servers.
```
Comment (bluemorbo) [inline source/network/load-balancers/components/access-controls.md:35]: It's worth mentioning that if a customer is looking for a specific behaviour that's not listed, that they should speak to their AM so we can look into adding it to the list of templates

File: source/network/load-balancers/components/target-groups.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Target Groups
+
+Target groups are a collection of target servers that will receive and process requests to the load balancer.
+
+To add your target groups, first go to the load balancer you want to add the target group too. Then click `Target Groups` on the tabs at the top of the screen, followed by `Create Target Group`.
```
Comment (bluemorbo) [inline source/network/load-balancers/components/target-groups.md]:
```suggestion
To add your target groups, first go to the load balancer you want to add the target group to. Then click `Target Groups` on the tabs at the top of the screen, followed by `Create Target Group`.
```

File: source/network/load-balancers/components/target-groups.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+## Load Balancing Methods
+
+There are 3 ways to balance traffic between the target servers, know as the load balancing method:
+* **Round Robin:** Sends traffic first to server A, then to server B, then to server A and so on. This has a slightly lower overhead but can lead to hot spots developing if sticky sessions are in use.
+* **Least Connections:** Intelligently sends traffic to the target server that has the least connections when the request is received
+* **Source:** The source IP address of the request is hashed and then this is used to determine which target server should be used. It ensures that the IP will always end up on the same target server providing no servers are added or removed.
+
+## Timeouts
+
+The load balancer will wait so long before a request timeouts. In this section, you are able to configure how long this delay is if the default values are not suited to your use.
```
Comment (bluemorbo) [inline source/network/load-balancers/components/target-groups.md]: Should this be "times out"?

**Comment by bluemorbo** [comment]
Few quick tweaks, and there seems to be a "spelling mistake" (probably a technical term we've not yet identified) on the Getting Started file.

I'll check everything links properly, but this is almost g2g

**Comment by rbibby** [comment]
> Few quick tweaks, and there seems to be a "spelling mistake" (probably a technical term we've not yet identified) on the Getting Started file.
> 
> I'll check everything links properly, but this is almost g2g

This is the html file extension on the links, tried to add ignore tags to it last time which didn't work so it was merged anyway.

**Comment by rbibby** [comment]
@bluemorbo I've made my changes

**Review by bluemorbo — APPROVED** [review]

**Comment by bluemorbo** [comment]
@miff2000 spell check aside, this is good to go :+1:

**Review by miff2000 — APPROVED** [review]

**Pull request merged**